spec/ics-2-consensus-requirements/README.md

```diff
@@ -0,0 +1,124 @@
+---
+ics: 2
+title: Consensus Verification
+stage: proposal
+category: ibc-core
+requires: 23
+required-by: 3
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-03-05
+---
+
+## Synopsis
+
+Consensus verification requires the properties that the chains on the network are 
+expected to satisfy. The properties are needed for efficient and safe verification on the 
+higher level mechanics, such as connection and channel semantics. The algorithm who uses these 
+properties to verify another chain is referred as "lightclient", which is embedded on the chains.
+
+## Specification
+
+### Motivation
+
+`Header` defines required properties of the blockchain on the network. The implementors can 
+check whether the consensus that they are using is qualified to be connected to the network or 
+not. If not, they can modify the algorithm or wrap it with additional logic to make it 
+compatible with the specification. It also provides base layer for the protocol that the other 
+components can rely on.
+
+### Desired Properties
+
+* Blockchains, defined as an infinite list of `Commit` starting from its genesis, is linear; no 
+conflicting `Commit`s can be both validated, thus no data can be rewritten after it has been 
+committed. Two `Commit`s are conflicting when both has same height but not equal.
+
+* Verifiers can verify future `Commit`s using an existing `TrustedCommit`. When the verifier 
+validates it, the verified header is in the canonical blockchain.
+
+* `TrustedCommit`s contains an accumulator root(ICS23) that the other logics can verify whether 
+key-value pairs exists or not with it.
+
+### Technical Specification
+
+#### Definitions
+
+* `TrustedCommit` is a blockchain commit which can be used to prove future commits, stored in
+  one blockchain to verify the state of the other.
+  Defined as 3-tuple `(v :: Commit -> (Error|TrustedCommit), r :: AccumulatorRoot)`, where
+    * `v` is the verifier, proves child `Commit.p` and returns the updated `TrustedCommit`
+    * `r` is the `AccumuatorRoot`, used to prove internal state
+
+* `Commit` is a blockchain header which provides information to update `TrustedCommit`, 
+  submitted to one blockchain to update the stored `TrustedCommit`.
+  Defined as 2-tuple `(p :: CommitProof, tc :: TrustedCommit`, where
+    * `p` is the commit proof used by `TrustedCommit.v` to verify
+    * `tc` is the new `TrustedCommit` which will replace the existing one after the verification
+ 
+* `Consensus` is a blockchain consensus algorithm which generates valid `Commit`s.
```

Comment:
I don't understand this definition, can you explain? I think the below section is better.